fix: add steps to create a new AWS S3 bucket (#214)
elevenlabs/elevenlabs-docs: api-reference/how-to-use-tts-with-streaming-in-python.mdx

  <img src="/api-reference/images/aws_region_name.png" />
</Frame>

If you do not have an AWS S3 bucket, you will need to create a new one by following these steps:

> 1. Access the S3 dashboard: You can find S3 under "Storage" on the services menu. The S3 dashboard manages the access to your AWS services securely.

Reviewer: ```suggestion
1. Access the S3 dashboard: You can find S3 under "Storage" on the services menu.
```

<Frame caption="AWS S3 Dashboard">
  <img src="/docs/api-reference/images/aws_s3_dashboard.png" />
</Frame>

2. Create a new bucket: On the S3 dashboard, click the "Create bucket" button.

<Frame caption="Click Create Bucket Button">
  <img src="/docs/api-reference/images/aws_s3_create_bucket.png" />
</Frame>

> 3. Enter a different bucket name: Populate the input field with a fresh bucket name. Afterwards, click on the "Create bucket" button. The newly added bucket will appear in the list.

Reviewer: ```suggestion
3. Enter a bucket name and click on the "Create bucket" button. You can leave the other bucket options as default. The newly added bucket will appear in the list.
```
